pkg/workflow/controllers/chaos_node_reconciler.go

```diff
@@ -51,51 +49,32 @@ func (it *ChaosNodeReconciler) Reconcile(request reconcile.Request) (reconcile.R
 		return reconcile.Result{}, client.IgnoreNotFound(err)
 	}
 
+	if !v1alpha1.IsChoasTemplateType(node.Spec.Type) {
+		return reconcile.Result{}, nil
```

Comment:
Should we return an error here?

Reply:
No, this reconciler would receive events about all workflownode, event type is not `Type<Some>Chaos`, and the `v1alpha1.IsChoasTemplateType()` plays as a filter.